### DISCUSSION TITLE
About the license

### DISCUSSION THREAD

**Discussion by kMutagene** [fslaborg/FSharp.Stats#110] [Ideas]
If i am not mistaken, one of the main reasons for apache 2.0 was the bundling with MSF right? so could we now move to MIT?

**Comment by smoothdeveloper** [comment]
@kMutagene as far as I know, apache 2.0 > MIT license.

https://www.gnu.org/licenses/license-list.en.html#Expat

> For substantial programs it is better to use the Apache 2.0 license since it blocks patent treachery.
